hotshot-state-prover/src/v1/snark.rs

```diff
@@ -159,18 +158,12 @@ mod tests {
             cur *= &beta;
         }
 
-        let window_size = FixedBase::get_mul_window_size(max_degree + 1);
-
-        let scalar_bits = <Bn254 as Pairing>::ScalarField::MODULUS_BIT_SIZE as usize;
         let g_time = start_timer!(|| "Generating powers of G");
-        // TODO: parallelization
-        let g_table = FixedBase::get_window_table(scalar_bits, window_size, g);
-        let powers_of_g = FixedBase::msm::<<Bn254 as Pairing>::G1>(
-            scalar_bits,
-            window_size,
-            &g_table,
-            &powers_of_beta,
-        );
+        let powers_of_g = g
+            .batch_mul(&powers_of_beta)
```

Comment:
btw, recently i learned that `batch_mul()` internally does two things: 1) build a lookup table, 2) does batched mul
https://github.com/EspressoSystems/timeboost/pull/456#issuecomment-3209054951

so it makes little sense to use this api directly if we only batch once. because each time will incur a "build preprocessing table" high cost. 

a simple parallelization of `.mul()` is actually faster.
(again, feel free to fix this later)